proofreader/__main__.py

```diff
@@ -1,15 +1,18 @@
-"""Main module for proofreader.
-
-This module
-
-- Takes `n` commandline arguments as `targets` and calls the runner.run function
-
-"""
+import argparse
 import os
-import sys
 
 from proofreader.runner import run
 
 DIRECTORY = os.getcwd()
 
-run(targets=sys.argv[1:], config_dir=DIRECTORY)
+PARSER = argparse.ArgumentParser(description='proofreader')
+
+PARSER.add_argument('--check-licenses', type=bool, help='Check for supported licenses .e.g. true')
+PARSER.add_argument('--targets', default=[], nargs='*',
```

Comment:
why not just insist on at least one target (`nargs='+'`) here? 

it would save you the `if targets: ...` in `runner`

Reply:
Good spot. I did wrestle with this briefly, but not insisting on at least one arg enabled you to use the license checking independently if required, that flexibility justified the addition of the `if targets:` in my opinion.

Reply:
no worries